src/main/java/Eliz/Eliz.java

```diff
@@ -0,0 +1,331 @@
+                    System.out.println(t);
+                    System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+                    /*every time an object is added or edited, append to file*/
+                    try {
+                        writeToFile(filePath, tasks.get(tasks.size()-1).getTaskType() + "|"
+                                + tasks.get(tasks.size()-1).getStatusIcon() + "|"
+                                + tasks.get(tasks.size()-1).getDescription() + System.lineSeparator());
+                    } catch (IOException e) {
+                        System.out.println("Something went wrong: " + e.getMessage());
+                    }
+                    break;
+                }
+            }
+        }
+//        if (breakTaskNames[0].equalsIgnoreCase("todo") || breakTaskNames[0].equalsIgnoreCase("deadline")
+//                || breakTaskNames[0].equalsIgnoreCase("event")) {
+//            /** add line to todo, deadline, or event by creating the respective object */
+//            Task t = createTask(line);
+//            tasks.add(t);
+//            System.out.println("Got it. I've added this task ");
+//            System.out.println(t);
+//            System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+//
+//            /*every time an object is added or edited, append to file*/
+//            try {
+//                writeToFile(filePath, tasks.get(tasks.size()-1).getTaskType() + "|" + tasks.get(tasks.size()-1).getStatusIcon() + "|"
+//                        + tasks.get(tasks.size()-1).getDescription() + System.lineSeparator());
+//            } catch (IOException e) {
+//                System.out.println("Something went wrong: " + e.getMessage());
+//            }
+//
+//        } else {
+//            if (line.equalsIgnoreCase("list")) { //check if action is t
+//                //echo or print tasks
+//                printTasks(tasks);
+//            } else if (line.contains("mark")) { //to check if todos are marked
+//                Eliz.markOrUnmark(line, tasks);
+//            } else {
+//                System.out.println("OOPS!!! I'm sorry but I do not understand what you mean");
+//                throw new ElizException();
+//            }
+//        }
+//        if (!line.equalsIgnoreCase("list") && !line.contains("mark") && breakTaskNames.length < 2) {
+//            System.out.println("OOPS!!! The description of a " + breakTaskNames[0] + " cannot be empty.");
+//            throw new ElizException();
+//        }
+
+    }
+
+    public static void deleteFileBeforeWriting(String filePath) throws IOException{
+        Files.delete(Paths.get(filePath));
+    }
+
+    public static void createTasksFromFileContents(String filePath, ArrayList<Task> tasks) throws FileNotFoundException {
+        File f = new File(filePath); //helps to create a file for the give file path
+        Scanner s = new Scanner(f); //create scanner using file as a source
```

Comment:
You could inserted comments on separate line with same indentation relative to the code. This will not break the logical structure of the code. See [commenting conventions](/https://se-education.org/guides/conventions/java/basic.html#comments).